e2e/test/scenarios/native-filters/sql-field-filter-string.cy.spec.js

```diff
@@ -60,13 +67,18 @@ describe("scenarios > filters > sql filters > field filter > String", () => {
         FieldFilter.openEntryForm({ isFilterRequired: true });
 
         searchTerm
-          ? FieldFilter.pickDefaultValue(searchTerm, value)
-          : FieldFilter.addDefaultStringFilter(value);
+          ? FieldFilter.pickDefaultValue(searchTerm, value, {
+              buttonLabel: "Update filter",
+            })
+          : FieldFilter.addDefaultStringFilter(value, {
+              buttonLabel: "Update filter",
+            });
 
         SQLFilter.runQuery();
 
         cy.get(".Visualization").within(() => {
           cy.findByText(representativeResult);
+          cy.findByText("Toucan").should("not.exist");
```

Comment:
This entire test suite was broken or, rather, not checking anything.

Prior to my updates, it:

1. typed a string (for example, "Synergistic Granite Chair") into the input
2. **did not** chose it from the list and clicked the "Add filter" button, which **did not do** anything
3. checked that there's a result in the table — but since it **wasn't even filtered** in the first place, each result was always there